packages/rancher-istio/charts/values.yaml

```diff
@@ -74,6 +77,15 @@ global:
     tag: 1.7.3
   defaultPodDisruptionBudget:
     enabled: true
+  defaultTolerations: []
+  defaultResources: {}
+  imagePullPolicy: IfNotPresent
+  imagePullSecrets: []
+  arch:
+    amd64: 2
+    s390x: 2
+    ppc64le: 2
+  defaultNodeSelector: {}
```

Comment:
If we are splitting chart out, all of this should be done in sub-chart, and then edited below under the tracing node. Under that node, we should only be setting things we are explicitly overriding.